seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -85,7 +85,7 @@
         ""
       ],
       "tail": [
-        "if(typeof myName !== \"undefined\"){(function(v){return v;})(myName);}"
+        "if(typeof myName !== \"undefined\"){ console.log(myName); }"
```

Comment:
Is this tail code necessary? This challenge, "Declare JavaScript Variables," just asks to declare a variable, without setting it equal to anything.